tests/test_git.py

```diff
@@ -205,7 +205,9 @@ def test_set_ref(tmp_dir: TmpDir, scm: Git, git: Git):
 def test_get_ref(tmp_dir: TmpDir, scm: Git, git: Git):
     tmp_dir.gen({"file": "0"})
     scm.add_commit("file", message="init")
+
     init_rev = scm.get_rev()
+
```

Comment:
this isn't a big deal, but try not to add extra whitespace changes for no reason